### ISSUE TITLE
Stackdriver exporter HTTP method not populated in Stackdriver Trace UI

### ISSUE THREAD

**Issue by nmbro** [open-telemetry/opentelemetry-dotnet#442]
When using the Stackdriver exporter - the HTTP Method in the Stackdriver Trace list is not being populated.

![stackdriver_issue](https://user-images.githubusercontent.com/3308346/72440518-4d0e3980-37a9-11ea-9bae-85b9d40c97de.PNG)

As you can see the top part of the trace list HTTP Method is empty - even though the http.method property is filled.

I'm probably using it wrong?

The version I'm using is: `0.2.0-alpha.148`

**Comment by MikeGoldsmith** [comment]
Hi @nmbro 

Please can you provide an example of how you're configuring OpenTelemetry and the StackDriver exporter?

Thanks

**Comment by nmbro** [comment]
I'm using Autofac for my IOC here's my extensionmethod

```csharp
applicationSettingsContainer.GetSetting("TracingUseZipkin") = ""
applicationSettingsContainer.GetSetting("TracingUseStackdriver") = "true"
applicationSettingsContainer.GetSetting("TracingStackdriverProjectId") = "example-project-name"
```

```csharp
public static void AddOpenTelemetry(this ContainerBuilder containerBuilder, string tracerName)
		{
			containerBuilder
				.AddSingleton<Sampler>(c => double.TryParse(c.Resolve<IApplicationSettingsContainer>().GetSetting("TracingSamplingPercent"),
					NumberStyles.Number, CultureInfo.InvariantCulture,
					out double samplingPercentage) ? (Sampler)new ProbabilitySampler(samplingPercentage) : (Sampler)new NeverSampleSampler());

			containerBuilder
				.AddSingleton<TracerFactory>(c =>
				{
					return TracerFactory.Create(b =>
					{
						b.SetSampler(c.Resolve<Sampler>());

						IApplicationSettingsContainer applicationSettingsContainer = c.Resolve<IApplicationSettingsContainer>();

						bool useZipkin = applicationSettingsContainer.GetSetting("TracingUseZipkin") == "true";
						string zipkinUrl = applicationSettingsContainer.GetSetting("TracingZipkinUrl");
						if (useZipkin && !string.IsNullOrEmpty(zipkinUrl))
						{
							b.UseZipkin(o =>
							{
								o.ServiceName = tracerName;
								o.Endpoint = new Uri(zipkinUrl);
							});
						}

						bool useStackdriver = applicationSettingsContainer.GetSetting("TracingUseStackdriver") == "true";
						string stackdriverProjectId = applicationSettingsContainer.GetSetting("TracingStackdriverProjectId");

						if (useStackdriver && !string.IsNullOrEmpty(stackdriverProjectId))
						{
							b.AddProcessorPipeline(p => p.SetExporter(new StackdriverTraceExporter(stackdriverProjectId)));
						}

						b.SetResource(
							new Resource(
								new Dictionary<string, object>
								{
									{"service.name", tracerName}
								}.AsEnumerable()));
					});
				});

			containerBuilder.AddScoped<Tracer>(c => c.Resolve<TracerFactory>().GetTracer(tracerName));
		}
```

**Comment by MikeGoldsmith** [comment]
Okay, so nothing out of the ordinary then. Thanks for confirming.

**Comment by MikeGoldsmith** [comment]
Looks like we need to create a specific attribute within the span that is sent to StackDriver. I've started to have a look into it.

Also, StackDriver API version we currently have is fairly out of date and should be updated.

1.0.0-beta02 (Aug 2019) vs 1.1.0 (Dec 2019)

**Comment by MikeGoldsmith** [comment]
These seem to be the labels Google Trace uses if you configure their ASP.NET middleware. I'll test out swapping some of them out.

**Comment by nmbro** [comment]
> 
> 
> These seem to be the labels Google Trace uses if you configure their ASP.NET middleware. I'll test out swapping some of them out.

Cool let me know if I can use them internally too - that would be cool; Also btw the Name attribute doesn't seem to hit Google Stackdriver either

**Comment by MikeGoldsmith** [comment]
I've pushed a PR that replaces four labels that there were conversions for.

We set the StackDriver Trace.DisplayName property to the OTel Span.Name property. eg when you call the span when creating it `tracer.StartActiveSpan("/getResource", out var span)`. I'm not sure what we could / should use instead.

**Comment by MikeGoldsmith** [comment]
This should be resolved now #453 has been merged - you can test by grabbing a recent MyGet build.

Please re-open if required. Thanks.
